test/basic.ts

```diff
@@ -0,0 +1,84 @@
+import t from 'tap'
+import { fileURLToPath } from 'url'
+import read from '../src/read.ts'
+import spawnRead from './fixtures/setup.ts'
+
+const __filename = fileURLToPath(import.meta.url)
```

Comment:
Consider making spawnRead take a file: url and just pass import.meta.url?

Though I guess that just moves the fileURLToPath call over a bit, one less thing to track the test itself.